### PR TITLE
Pamela resources profiling

### PR DESCRIPTION
**Pull request by parostatkiem**
**Description**

Changes proposed in this pull request:

- add evaluation/production profiles handling to Pamela chart
- adjust evaluation and production profile values basing on observations
- increase Console-backend memory limit to `200Mi` in production profile

**Related issue(s)**

### CODE REVIEW

File: resources/console/profile-evaluation.yaml [hunk 1]
```diff
@@ -23,3 +23,11 @@ backend:
...
       cpu: 50m
       memory: 80Mi
+pamela:
+  resources:
+    requests:
+      cpu: 8m
+      memory: 55Mi
+    limits:
+      cpu: 40m
+      memory: 150Mi
```
Comment (dariadomagala) [inline resources/console/profile-evaluation.yaml:33]: Weekly and nightly clusters use 180Mi all the time, 150 might be not enough

**Review by dariadomagala — COMMENTED** [review]
Also, Krzysiu suggested to also fix this issue:
https://kyma-community.slack.com/archives/CD11JLTTN/p1608465633047000

File: resources/console/profile-evaluation.yaml (hunk 1 above)
Comment (parostatkiem) [reply resources/console/profile-evaluation.yaml:33]: No, weekly uses 53MB normally, 63MB on peak...
image

File: resources/console/profile-production.yaml [hunk 2]
```diff
@@ -22,4 +22,12 @@ backend:
       memory: 40Mi
     limits:
       cpu: 50m
-      memory: 150Mi
+      memory: 200Mi
```
Comment (parostatkiem) [inline resources/console/profile-production.yaml:25]: @dariadomagala I raised this value. I assumed the man with the problem was using a production cluster.

File: resources/console/profile-evaluation.yaml (hunk 1 above)
Comment (dariadomagala) [reply resources/console/profile-evaluation.yaml:33]: You're right, I selected the wrong color on the graph, my bad. If it's 63, then 150 might be a bit too much. same with production profile. I think 200 would be enough there

Comment (parostatkiem) [reply resources/console/profile-evaluation.yaml:33]: 😕 so.. 200 or 150? I think `150Mi` would be perfect for the production profile and `100Mi` for the evaluation one. Do you agree?

Comment (dariadomagala) [reply resources/console/profile-evaluation.yaml:33]: "150 might be a bit too much" was about the evaluation profile, "200 would be enough there" was about the production profile 🙂 sorry for the confusion.
Those values that you wrote should be fine 🙂

File: resources/console/profile-production.yaml (hunk 2 above)
Comment (dariadomagala) [reply resources/console/profile-production.yaml:25]: as we discussed, this issue should be moved to a separate task after all, please remove this change :)

**Review by kazydek — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**